### PR TITLE
Allow custom messages to be passed to error

### PR DESCRIPTION
**Pull request by aleccohan**
Allow custom messages to be passed to error.

### CODE REVIEW

File: src/routes/Make.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ const Make: React.FC = () => {
 
   return (
     <Main name='make'>
-      { error && <Error/> }
+      { error && <Error drink={input} ></Error> }
```
Comment (ryelo) [inline src/routes/Make.tsx]:
```suggestion
      { error && <Error drink={input}/> }
```

File: src/utils/index.tsx [hunk 2]
```diff
@@ -11,3 +11,14 @@ export const getParams = (param: string):string => {
 export const normalizePrice = (price: number, exchangeRate: number, digitsPastDecimal: number):number => {
   return parseFloat((exchangeRate * price).toFixed(digitsPastDecimal)); // sub for current price later
 }
+
+// Checks drink and sends appropriate message depending on the drink
+export const checkDrinkError = (drink: string):string => {
+  switch (drink) {
+  case 'Alcohol':
+    return 'Drink it strait, its the fastest way to reach your goal!'
+
```
Comment (ryelo) [inline src/utils/index.tsx]: can you add a `break;` here

**Review by ryelo — CHANGES_REQUESTED** [review]

File: src/utils/index.tsx [hunk 3]
```diff
@@ -11,3 +11,28 @@ export const getParams = (param: string):string => {
 export const normalizePrice = (price: number, exchangeRate: number, digitsPastDecimal: number):number => {
   return parseFloat((exchangeRate * price).toFixed(digitsPastDecimal)); // sub for current price later
 }
+
+// Checks drink and sends appropriate message depending on the drink
+export const checkDrinkError = (drink: string):string => {
+  switch (drink) {
+  case 'Alcohol':
+    return 'Drink it strait, its the fastest way to reach your goal!'
```
Comment (SteveHNH) [inline src/utils/index.tsx]: is it `strait` or `straight`?

Comment (aleccohan) [reply src/utils/index.tsx]: 🤦 Yup!  Straight

**Review by ryelo — APPROVED** [review]

**Review by SteveHNH — APPROVED** [review]

**Pull request merged**